### PR TITLE
Init models

### PR DESCRIPTION
**Pull request by dtmhdev89**
init models
rubocop_init-models

### CODE REVIEW

**Comment by dtmhdev89** [comment]
ready

File: app/models/product.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Product < ApplicationRecord
+  enum size: {no_size: 0, S: 1, M: 2, X: 3, XL: 4, XXL: 5}
+  enum color: {no_color: 0, white: 1, black: 2}
```
Comment (VuThiTranVan) [inline app/models/product.rb]: line code này sau lien code quan hệ  nhé

Sắp xếp lại thứ tự line code theo coding convention anh nhé
https://github.com/framgia/coding-standards/blob/master/vn/rails/standard.md#activerecord

**Comment by VuThiTranVan** [comment]
commented
fix lẹ anh nhé

**Comment by dtmhdev89** [comment]
ready

**Pull request merged**